fe/fe-core/src/main/java/com/starrocks/analysis/RevokeRoleStmt.java

```diff
@@ -0,0 +1,24 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+// REVOKE Role 'role' FROM 'user'
+// share the same parameter and check logic with GrantRoleStmt
+public class RevokeRoleStmt extends GrantRoleStmt {
```

Comment:
This can be done by one class with an enum to differ different statement?

Reply:
I thought about this, but to name a class `GrantRevokeStmt` or a parameter `isGrant` could be much confusion. Furthermore, it will bring in a lot of branches with if/else, which only makes it harder to understand. Considering these, I think adding a new class is not such a heavy implementation.